src/main/java/no/nav/tag/sykefravarsstatistikk/api/virksomhetsklassifikasjoner/SektorMappingService.java

```diff
@@ -0,0 +1,65 @@
+package no.nav.tag.sykefravarsstatistikk.api.virksomhetsklassifikasjoner;
+
+import no.nav.tag.sykefravarsstatistikk.api.domene.virksomhetsklassifikasjoner.Sektor;
+import no.nav.tag.sykefravarsstatistikk.api.enhetsregisteret.InstitusjonellSektorkode;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class SektorMappingService {
+    private static final Map<String, String> mapFraSSBSektorKodeTilInstitusjonellSektorkode;
+    static {
+        Map<String, String> map = new HashMap<>() {{
```

Comment:
Har aldri verifisert eller bevist det selv, men man kan lese her og der at _double brace initialization_ kan skape _memory leaks_
Kan f.eks erstattes med (mer verbose):
` Map<String, String> map =
                Map.of(
                        "1110", "3",
                        "1120", "3"
                );
`

Reply:
Map.of tar ikke så mange entries :P Men all right, jeg endrer på det!